gemfiles/rails_head.gemfile

```diff
@@ -2,6 +2,6 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails'
```

Comment:
I think this was fine as it was, it's the gemfile for the unreleased version of rails

Reply:
Was getting this error both in CI and locally
![image](https://user-images.githubusercontent.com/26125246/127682630-46276e3a-a066-464e-a5c0-064cd9c2505e.png)

Reply:
yes that's because we'd need to make the dependency on rails be more flexible (maybe just > x and not set a max version?). Otherwise you can just skip that test for now, but the gemfile is fine as it is